src/gui/windows/healthbar.zig

```diff
@@ -46,24 +46,24 @@ pub fn render() void {
 	var y: f32 = 0;
 	var x: f32 = 0;
 	var health: f32 = 0;
-	while(health < main.game.Player.maxHealth) : (health += 1) {
-		if(x >= window.contentSize[0]) {
+	while (health < main.game.Player.maxHealth) : (health += 1) {
```

Comment:
you still seem to have lots of unintended whitespace changes, probably still from when you used the formatter.

Reply:
I did the best I can do. I have to go now

Reply:
Looks good now, thanks.